### PR TITLE
Reset subset for a filter panel does not reset global spatial extent

### PR DESCRIPTION
**Pull request by anguss00**

### CODE REVIEW

**Comment by anguss00** [comment]
Changes made @danfruehauf

File: web-app/js/portal/filter/ui/FilterGroupPanel.js [hunk 1]
```diff
@@ -281,12 +281,19 @@ Portal.filter.ui.FilterGroupPanel = Ext.extend(Ext.Container, {
...
         Ext.each(this.filterPanels, function(panel) {
-            panel.handleRemoveFilter();
+            if (!that._isSpatialConstraint(panel)) {
+                panel.handleRemoveFilter();
+            }
         });
         this._updateLayerFilters();
     },
 
+    _isSpatialConstraint: function(panel) {
```
Comment (danfruehauf) [inline web-app/js/portal/filter/ui/FilterGroupPanel.js]: I meant something along the lines of `_isGlobal` or something like that.

Comment (anguss00) [inline web-app/js/portal/filter/ui/FilterGroupPanel.js]: So just rename the function then? Is the logic here OK?

Comment (danfruehauf) [inline web-app/js/portal/filter/ui/FilterGroupPanel.js]: Logic is fine. Expect it to become something with an `||` condition as the code progress (or doesn't).

**Comment by anguss00** [comment]
Renamed here @danfruehauf

**Comment by danfruehauf** [comment]
:+1:

**Comment by anguss00** [comment]
Bewdy mate

**Pull request merged**